sark/code/line.py

```diff
@@ -180,9 +180,19 @@ def endEA(self):
     def disasm(self):
         """Line Disassembly"""
         return idc.GetDisasm(self.ea)
+    @property
+    def type(self):
+        """return the type of the Line """
+        properties = {self.is_code:"code",
+                      self.is_data:"data",
+                      self.is_string:"string",
+                      self.is_tail:"tail",
+                      self.is_unknown:"unknown"}
+        for k, v in properties.items():
+            if k: return v
 
     def __repr__(self):
-        return "[{:08X}]    {}".format(self.ea, self.disasm)
+        return "[{:08X}]    {} {}".format(self.ea, self.disasm, self.type)
```

Comment:
If you are using a script, you should query the object, not parse text.
I think the `__repr__` is mainly for shell use, so the type is not relevant here.